access-ocp-api-through-cis.md

```diff
@@ -0,0 +1,104 @@
+## Accessing the api endpoint deployed on OpenShift through CIS
```

Comment:
We need more context here and possibly links to this topic from other places in the module.

- Why is this topic here? When and why do developers need to access the API endpoint?
- Does this apply in some specific use of the module? As in, `If you 


Is this what this set of steps is about? https://cloud.ibm.com/docs/openshift?topic=openshift-ingress-domains&interface=ui#ingress-domain-cis. If so, does it offer more context and we should link to it?

Reply:
Cloud Internet Services works as an interface - to access the applications deployed on OCP, VSI etc. As such standalone CIS cant be used. So giving this information, will complete the module.

https://cloud.ibm.com/docs/openshift?topic=openshift-ingress-domains&interface=ui#ingress-domain-cis - it is different than what we are talking about.